src/components/map/Map.tsx

```diff
@@ -113,33 +104,29 @@ export const Map = ({
   onMaximizeButtonPress,
   onMyLocationButtonPress,
   selectedMarker,
+  showsUserLocation = false,
   style,
   updatedRegion,
   ...otherProps
 }: Props) => {
   const { globalSettings } = useContext(SettingsContext);
   const { settings = {} } = globalSettings;
-  const { zoomLevelForMaps = {}, locationService = {} } = settings;
   const { locationSettings } = useLocationSettings();
+  const refForMapView = useRef<MapLibreGL.MapView>(null);
+  const cameraRef = useRef<MapLibreGL.Camera>(null);
+  const [clusters, setClusters] = useState<any[]>([]);
+  const [zoomLevel, setZoomLevel] = useState<number>(14);
+  const [isInitialFit, setIsInitialFit] = useState<boolean>(true);
```

Comment:
what is initial fit for?

Reply:
as the map opens with a set camera, wich it does when markers are available, the initialfit of the camera gets overwritten so the user is able to navigate through the map freely.
without initialfit the camera stays set, so as you open a map with markers and try to move the mapview gets set back to the initial camera view the map was initiated in.

Reply:
ok strange or do i get that wrong? which scenario can exist where this is intended?

Reply:
that was how the app behaved with the code until I introduced initial fit, so with `initialFit` after the camera gets set we can navigate freely without getting set back to the cameras initial view if we try to navigate/swipe throughout the map

Reply:
ok strange? which scenario can exist where this is intended when users move and it jumps back?

Reply:
maybe a scenario where the mapmarker should always be in the main focus and not in a draggable map, but since it's not our case we got the solution with setting and removing the initial fit